.github/tests.yaml

```diff
@@ -0,0 +1,12 @@
+broker[docker]: "tests/foreman/ -v -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'"
+deepdiff: "tests/foreman/endtoend/test_api_endtoend.py -v -k 'test_positive_get_links'"
+dynaconf[vault]: "tests/foreman/ -v -m 'build_sanity'"
+manifester: "tests/foreman/api/test_subscription.py -v -k 'test_positive_create_after_refresh'"
+navmazing: "tests/foreman/ui/test_repository.py -v -k 'test_positive_create_as_non_admin_user'"
+pyotp: "tests/foreman/ui/test_ldap_authentication.py -v -k 'test_positive_login_user_password_otp'"
+pytest-xdist: "tests/foreman/ -v -m 'build_sanity'"
```

Comment:
Run this test with multiple(At least2) workers as the xDist is to distribute tests across workers.

Reply:
Could be a good candidate as the xdist params of number of workers are being passed to the pytest.main() running in process test-run.

Reply:
As all unit tests are run as the process of the GitHub CI checks, I would rather run the sanity with -n 2, 4 just for getting into actual focus on the tests from the framework/application.